package.json

```diff
@@ -4,7 +4,9 @@
   "description": "<a name=\"readme-top\"></a>",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack serve",
+    "build": "webpack"
   },
```

Comment:
- [ ] It is required that the `scripts` in your package.json should something similar to this :point_down: 

```
     "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```
There's a missing `start` in your configuration. Kindly add that so that your can fire off on the browser without the need to visit the url `http://localhost:9000/`.